src/corppa/poetry_detection/core.py

```diff
@@ -198,6 +200,33 @@ def strip_whitespace(self) -> "Excerpt":
             ppa_span_text=self.ppa_span_text.strip(),
         )
 
+    def correct_page_excerpt(self, page_text: str) -> "Excerpt":
```

Comment:
Any thoughts on how we would call this outside of the Excerpt class context? e.g. if I want to run in batches on a polars dataframe?

No need to refactor now, but still worth thinking about